### DISCUSSION TITLE
A quick question about Qdrant implementation of VectorBase

### DISCUSSION THREAD

**Discussion by sunxiaoguang** [langgenius/dify#5599] [General]
Hey, I'm looking at the VectorBase interface and the Qdrant implementation of it. I'm just curious, why is the delete_by_document_id method missing in the Qdrant implementation?

Another quick question related to Qdrant integration is that I didn't see the GROUP_KEY being used in other integrations, only in Qdrant. Thanks

**Comment by JohnJyong** [comment]
Thanks for your feedback @sunxiaoguang 
1.   `delete_by_document_id`  this method is deprecated and I will remove this method in subsequent code
2.  By `GROUP_KEY` , Qdrant will index vectors for each user independently, significantly accelerating the process， this is the qdrant‘s feature.

**Reply by sunxiaoguang** [reply]
Cool, thanks for answering the questions. I really appreciate your answers.

**Comment by JohnJyong** [comment]
the deprecated method has deleted : https://github.com/langgenius/dify/pull/5612
